docs/websites/ror/ror-with-unicorn-and-nginx-on-ubuntu-14-04.md

```diff
@@ -0,0 +1,211 @@
+---
+author:
+    name: Linode Community
+    email: docs@linode.com
+description: 'Setup Ruby on Rails stack on Ubuntu 14.04 using Nginx and Unicorn'
+keywords: 'ruby on rails,unicorn rails,ruby on rails ubuntu 14.04'
+license: '[CC BY-ND 3.0](http://creativecommons.org/licenses/by-nd/3.0/us/)'
+published: 'Saturday, December 19th, 2015'
+modified: Saturday, December 19th, 2015
+modified_by:
+    name: Linode
+title: 'How to deploy Ruby on Rails applications with Unicorn and Nginx on Ubuntu 14.04'
+contributor:
+    name: Vaibhav Rajput
+    link: https://twitter.com/rootaux
+external_resources:
+ - '[Ruby on Rails](http://rubyonrails.org/)'
+---
+
+Ruby on Rails is a popular web-application framework that allows developers to create a dynamic web application. This guide describes how to deploy Rails applications on server using Unicorn and Nginx on Ubuntu 14.04.
+
+Unicorn is an HTTP server, just like Passenger or Puma. Since Unicorn cannot be accessed by users directly we will be using Nginx as the reverse proxy that will buffer requests and response between users and Rails application.
+
+Before starting this guide, make sure that  you have read through and completed our [Getting Started](/docs/getting-started#debian-7--slackware--ubuntu-1404) and [Securing Your Server](/docs/security/securing-your-server/) guides.
+
+##Set the hostname
+
+1.  Before you install any package, ensure that your hostname is correct by completing the [Setting Your Hostname](/docs/getting-started#sph_setting-the-hostname) section of the Getting Started guide. Issue the following commands to verify that hostname:
+
+    hostname
+    hostname -f
+
+##System Setup
+
+2.  Make sure your system is up to date using apt:
+
+    sudo apt-get update && apt-get upgrade
+
+This ensures that all software is up to date and running the latest version.
+
+##Install Ruby
+
+1. Install Ruby dependencies using APT:
+
+    sudo apt-get install git-core curl zlib1g-dev build-essential libssl-dev libreadline-dev libyaml-dev libsqlite3-dev sqlite3 libxml2-dev libxslt1-dev libcurl4-openssl-dev python-software-properties libffi-dev nodejs
+
+2. Download the latest version of Ruby. At the time of writing this article, current latest and stable version is 2.2.3:
+
+	wget https://cache.ruby-lang.org/pub/ruby/2.2/ruby-2.2.3.tar.gz
+
+3. Unpack the tarball:	
+
+	tar -xzvf ruby-2.2.3.tar.gz
+
+4. Move to the extracted directory:
+
+	cd ruby-2.2.3.tar.gz
+
+5. Compile the Ruby:	
+
+	./configure
+
+6. Getting ready for the installation:
+
+	make
+
+7. Issue the following command and it will install Ruby:
+
+	sudo make install
+
+##Install Rails
```

Comment:
Clearly there shouldn't be an entire section just for one command. How about merging with the previous section?